src/renderers/dom/client/utils/DOMLazyTree.js

```diff
@@ -56,8 +56,11 @@ var insertTreeBefore = createMicrosoftUnsafeLocalFunction(
     // DocumentFragments aren't actually part of the DOM after insertion so
     // appending children won't update the DOM. We need to ensure the fragment
     // is properly populated first, breaking out of our lazy approach for just
-    // this level.
-    if (tree.node.nodeType === 11) {
+    // this level. Also, some <object> plugins (like Flash Player) will read
+    // <param> nodes immediately upon insertion into the DOM, so <object>
+    // must also be populated prior to insertion into the DOM.
+    if (tree.node.nodeType === 11 ||
+        tree.node.nodeName && tree.node.nodeName.toLowerCase() === 'object') {
```

Comment:
Can we do `tree.node.nodeType === 1 && tree.node.nodeName === 'OBJECT' && (tree.node.namespaceURI == null || tree.node.namespaceURI === DOMNamespaces.html)`? Otherwise I suppose I'm good with this.